### PR TITLE
Updates the landing page

### PR DESCRIPTION
**Pull request by hyounes4560**
Changes the location of the talks section from the landing page
Adds an overview paragraph to the landing page
Updates the level of few section headings in the Getting Started landing page
Changes the order of talks by time
Other minor edits

### CODE REVIEW

File: content/docs/_index.md [hunk 1]
```diff
@@ -4,24 +4,30 @@ weight=1
...
+
+The CNB project was initiated by Pivotal and Heroku in January 2018 and joined the Cloud Native Computing Foundation (CNCF) as an Apache-2.0 licensed project in October 2018. It is currently an incubating project within the CNCF.
 
 ---
 
 ## Tutorials
 
 * [An App’s Brief Journey from Source to Image](/docs/app-journey/) - An easy-to-follow introduction to Cloud Native Buildpacks using `pack`, a command line tool for Cloud Native Buildpacks.
-* [Creating a Cloud Native Buildpack](/docs/for-buildpack-authors/tutorials/basic-buildpack) - Tutorial walking through the creation of a simple Ruby buildpack.
+* [How to write a basic buildpack](/docs/for-buildpack-authors/tutorials/basic-buildpack) - A tutorial walking through the creation of a simple Ruby buildpack.
```
Comment (AidanDelaney) [inline content/docs/_index.md:20]:
```suggestion
* [How to write a basic buildpack](/docs/for-buildpack-authors/tutorials/basic-buildpack) - A tutorial walking through the creation of an example NodeJS buildpack.
```

I personally dislike the word "simple".  When I read that something should be "simple" and if I don't understand it, then I feel like an idiot.

**Review by AidanDelaney — APPROVED** [review]
I have one suggested change.  Otherwise, LGTM!

**Review by natalieparellano — APPROVED** [review]
Love it. Thanks for doing this @hyounes4560 🙏🏼

File: content/docs/_index.md (hunk 1 above)
Comment (hyounes4560) [reply content/docs/_index.md]: Updated

**Comment by AidanDelaney** [comment]
I've manually reviewed the DCO. Everything looks fine to me.  I'm not sure why our DCO check is failing to return.  We're using the DCO app at https://probot.github.io/apps/dco/

**Pull request merged**